.circleci/config.yml

```diff
@@ -0,0 +1,103 @@
+---
+version: 2.1
```

Comment:
We're slowly moving away from CircleCI as the standard build pipeline.

There's a new CI workflow in `prometheus/prometheus`.

Reply:
Will do!